src/index.scss

```diff
@@ -42,3 +42,4 @@
 @import './wizard/index';
 @import './skip-to-content/index';
 @import './surface/index';
+@import './radio-tile/index';
```

Comment:
Would be nice to keep the alphabetical order here.. I see, 2 last additions did not do that :(